### PR TITLE
Improve French translation

### PR DESCRIPTION
**Pull request by reda-alaoui**

### CODE REVIEW

**Comment by jonkoops** [comment]
@reda-alaoui could you get another French native speaker to review your PR?

**Comment by reda-alaoui** [comment]
@jonkoops One of my colleague can do that if you want. I don't where to find one otherwise.

**Comment by jonkoops** [comment]
That is fine, it always good to have another set of eyes on this. If your colleague can review it that would be great!

File: themes/src/main/resources-community/theme/base/login/messages/messages_fr.properties
Comment (roman-garcia) [inline themes/src/main/resources-community/theme/base/login/messages/messages_fr.properties:401]: sans mots de passe => sans mot de passe

File: themes/src/main/resources-community/theme/base/login/messages/messages_fr.properties
Comment (roman-garcia) [inline themes/src/main/resources-community/theme/base/login/messages/messages_fr.properties:414]: l''authentication produit => l''authentication produite

**Review by roman-garcia — CHANGES_REQUESTED** [review]

**Review by roman-garcia — APPROVED** [review]

**Review by jonkoops — APPROVED** [review]

**Comment by reda-alaoui** [comment]
@jonkoops are we waiting for another reviewer?

**Comment by jonkoops** [comment]
Yes, we need a maintainer to approve this. I'll see if one is available to do so.

**Review by ssilvert — APPROVED** [review]

**Pull request merged**
